.gitignore

```diff
@@ -0,0 +1,2 @@
+/composer.lock
+/vendor
```

Comment:
missing trailing /:
`/vendor/`

Reply:
there seem to be no consistent way of doing this across @symfonycorp or @symfony, some repositories use `vendor/`, others `/vendor`, and others `/vendor/`

should we fix this across all our repositories ?

Reply:
What you really want to achieve is `/vendor/` :)